src/en/roliascan/src/eu/kanade/tachiyomi/extension/en/roliascan/RoliaScan.kt

```diff
@@ -0,0 +1,323 @@
+package eu.kanade.tachiyomi.extension.en.roliascan
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.POST
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import okhttp3.FormBody
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import rx.Observable
+import uy.kohesive.injekt.injectLazy
+
+// Theme: AnimaCEWP
+class RoliaScan : ParsedHttpSource() {
+
+    override val name = "Rolia Scan"
+
+    override val baseUrl = "https://roliascan.com"
+
+    override val lang = "en"
+
+    override val supportsLatest = true
+
+    private val json: Json by injectLazy()
+
+    override val client = network.cloudflareClient.newBuilder()
+        .rateLimit(3)
+        .build()
+
+    // ======================== Popular ======================================
+    private val popularFilter by lazy {
+        FilterList(SelectionList("", listOf(ratingList.maxBy { it.value })))
+    }
```

Comment:
no need for lazy

Reply:
> no need for lazy

In this case, it's needed because ```ratingList``` gets initialized after ```popularFilter```. If I take it out, I'll have to move ```ratingList``` before ```popularFilter``` is initialized, pulling it out from the 'Filters' section.